### PR TITLE
misc: fix roundtrip

### PR DESCRIPTION
**Pull request by paulirish**
just ran `yarn update:sample-json` on master and this is the diff i got. 

curious if everyone else has the same changes?

### CODE REVIEW

**Comment by Hoten** [comment]
node version?

**Comment by paulirish** [comment]
node --version
v11.13.0

**Comment by Hoten** [comment]
I get the same w/ v10.14.2

**Comment by paulirish** [comment]
lgtm me then ? :)

**Comment by brendankenny** [comment]
can confirm I get the same diff on 11.13.0 as well

File: proto/sample_v2_round_trip.json [hunk 1]
```diff
@@ -2141,91 +2141,91 @@
...
+                ], 
+                "type": "table"
+            }, 
+            "displayValue": "18 requests \u2022 157 KB", 
+            "id": "resource-summary", 
+            "score": null, 
+            "scoreDisplayMode": "informative", 
             "title": "Keep request counts low and transfer sizes small"
-        },
+        },
```
[the remark this answers is not in the document]
Comment (brendankenny) [reply proto/sample_v2_round_trip.json]: I fixed the trailing spaces in #8818, we could try to get rid of the useless `.0`s, but it's messier...

**Pull request merged**